xen/xenops_types.ml

```diff
@@ -84,8 +114,14 @@ module Vm = struct
     boot_order = "";
     qemu_disk_cmdline = false;
     qemu_stubdom = false;
+    firmware = default_firmware;
+    nvram = Nvram.default_t;
```

Comment:
There seems to be no option for not having nvram at all now. Don't we want to make this an option type?

Reply:
I'm a bit undecided whats the best way here, but since we have the Nvram.default_t I'll make this an option type, and if its missing then use the default only in HVM mode.